expr/sort.go

```diff
@@ -29,7 +29,7 @@ func isNull(val zng.Value) bool {
 // is governed by the nullsMax parameter.  If this parameter is true,
 // a record with a null value is considered larger than a record with any
 // other value, and vice versa.
-func NewSortFn(nullsMax bool, fields ...FieldExprResolver) SortFn {
+func NewCompareFn(nullsMax bool, fields ...FieldExprResolver) CompareFn {
```

Comment:
Nit: could also change the usage of `sorters`, `sorter`, `sf` , and other var names that are derived from 'sort' in this file as well.

Reply:
👍